docs/api-reference/index.md

```diff
@@ -14,17 +14,17 @@
    Cell
    CellInBeamLog
    Depolarized
-   DirectBeamBackgroundRegion
+   DirectBeamBackgroundQRange
    DirectBeamNoCell
    DirectBeamQRange
    Down
    He3CellLength
    He3CellPressure
    He3DirectBeam
    He3FillingTime
-   He3Opacity
-   He3Polarization
-   He3Transmission
+   He3OpacityFunction
+   He3PolarizationFunction
+   He3TransmissionFunction
```

Comment:
Especially for the functions where a fit has to be performed, I actually like the naming `...Function` --> it is more clear (to me) what is the function and what is the result itself

Reply:
I also think having "function" here is important, since we initially thought we would return the evaluated function. The comparison to `Variable` and `DataArray` is not equivalent, I think.